src/Makefile.Android

```diff
@@ -116,14 +116,14 @@ ifeq ($(ANDROID_ARCH),ARM)
     CFLAGS = -std=c99 -march=armv7-a -mfloat-abi=softfp -mfpu=vfpv3-d16
 endif
 ifeq ($(ANDROID_ARCH),ARM64)
-    CFLAGS = -std=c99 -target aarch64 -mfix-cortex-a53-835769
+    CFLAGS = -std=c99 -march=armv8-a -mfix-cortex-a53-835769
```

Comment:
This is the only change that I am unsure of and might need to be reverted.

Reply:
I was investigating this but I couldn't find enough information. In any case, I'm merging it but it requires some further testing to verify it works on other platforms.

Related info: https://man.archlinux.org/man/aarch64-linux-gnu-gcc.1.en#march=